lib/query-interface.js

```diff
@@ -637,8 +637,8 @@ class QueryInterface {
     );
   }
 
-  increment(instance, tableName, values, identifier, options) {
-    const sql = this.QueryGenerator.incrementQuery(tableName, values, identifier, options.attributes);
+  incrementOrDecrement(instance, tableName, values, identifier, options) {
```

Comment:
In `model.js`: 
- we keep the `decrement()` calling `this.increment()` ? or 
- we replicate the same code from `increment()` to `decrement()` ? or
- abstract the `increment()` internals to another method to be able to call `decrement()` in query-generator?